taxonomy-manager-rest-server/src/main/java/com/digirati/taxman/rest/server/taxonomy/ConceptModelRepository.java

```diff
@@ -115,4 +168,40 @@ public ConceptModel create(ConceptModel model) {
 
         return find(uuid);
     }
+
+    /**
+     * Analyze the changes between relationships and create symmetric relations if needed
+     *
+     * @param model The new data for the Concept
+     * @param existing (Can be null) The current version of the Concept, if one exists
+     */
+    private void applySymmetricRelationChanges(ConceptModel model, ConceptModel existing) {
+
```

Comment:
Unnecessary whitespace here, also can we run a formatter over this code? There's inconsistent spacing between characters, and arguments should be on the same line as the parameters:
```
BiConsumer<UUID, ConceptRelationshipType> createRelationship =
         (relatedUuid, relationshipType) -> {
         };
```